### ISSUE TITLE
Custom fonts iOS not working after upgrade to 6.2.1

### ISSUE THREAD

**Issue by siebeprojects** [react-native-community/react-native-svg#606]
Upgrade react-native-svg from: 5.5.1 to 6.2.1 

**Version 5.5.1**
For both Android and iOS the custom fonts work in the normal react-native Views.

However:
Android - Custom fonts do **not** work in react-native-svg Text elements.
iOS - Custom fonts do work correctly in react-native-svgText elements.

**Version 6.2.1**
For both Android and iOS the custom fonts continue to display fine in the normal react-native Views.

But in this version the behaviour with 5.5.1 has reversed.
Custom fonts now work in Android but do not work anymore in iOS in react-native-svg Text elements.

Did anything change for iOS given the handling of custom fonts ? 

If I uninstall 6.2.1 and install and link 5.5.1 again then the custom fonts work again in iOS but stop working in Android in react-native-svg Text elements.

**Comment by msand** [comment]
I'm unable to replicate, works when I test it. Could you post a full replication? 
What I can remember now, only the font handling in android has changed, to support custom fonts. Can't remember changing the logic in iOS for that.

**Comment by siebeprojects** [comment]
Thanks for the quick reply, if it is working for you it should work for me. 
Let me try again and I will get back when I know more.

**Comment by siebeprojects** [comment]
This is what is happening and here are the use cases.

1. If in XCode iOS app has installed one custom font
- Roboto-Light.ttf

The SVG Text element shows this font correctly with fontFamily="Roboto-Light"
Changing the fontFamily to "foo" shows the default system font.

2. iOS app has installed two custom fonts
- Roboto-Light.ttf
- Roboto-Medium.ttf

In both cases when 
fontFamily="Roboto-Light" or fontFamily="Roboto-Medium" 

the SVG Text element shows the text by always using the "Roboto-Medium" font. 
Changing the fontFamily to "foo" shows the text with the default system font.

So on iOS it identifies correctly the fontFamilies "Roboto-Light" and "Roboto-Medium".
However, it just always uses the "Roboto-Medium" font.

**Comment by zschiller** [comment]
@msand Not sure if this is related to this issue, but I am currently unable to use custom fonts imported with Expo: https://github.com/expo/expo/issues/1959

**Comment by msand** [comment]
Is this still an issue? Please comment or reopen if so.

**Comment by msand** [comment]
Custom fonts have been fixed, try with the latest version.

**Comment by nijikokun** [comment]
I am still seeing this as an issue with the latest Expo SDK (33) and latest (react-native-svg).

```js
    import * as Font from 'expo-font';
    await Font.loadAsync({
      Chewy: require('../fonts/Chewy-Regular.ttf')
    })
```

```js
const SvgProps = {
  width: '200',
  height: '64',
  viewBox: '0 0 100 100'
};

const SvgTextProps = {
  fill: '#000',
  stroke: '#FFF',
  strokeWidth: '2',
  fontSize: '48',
  fontWeight: 'bold',
  fontFamily: 'Chewy',
  x: '0',
  y: '74',
  textAnchor: 'middle'
};

        <Svg {...SvgProps}>
          <SVGText {...SvgTextProps}>{this.props.title}</SVGText>
        </Svg>
```

**Comment by Jujenji** [comment]
encountering the same as @nijikokun stated, I also can not use the fontFamily prop with a success. All other fontProps seem to work, only the family is not accounted for.  Using Expo SDK 33 and neither rn svg 9.4.0 nor 9.6.2 (using the former because of dependency issues of expo)

**Comment by msand** [comment]
Interesting, seems to work fine for me: https://github.com/msand/VariableFontWeightAnimation/blob/master/App.js

**Comment by msand** [comment]
Seems to work correctly now: https://github.com/msand/ExpoTestSvgFonts/blob/master/App.js

```jsx
import React, { Component } from 'react';
import { StyleSheet, View, Text } from 'react-native';
import { Svg, G, Text as SVGText } from 'react-native-svg';
import * as Font from 'expo-font';

StyleSheet.setStyleAttributePreprocessor('fontFamily', Font.processFontFamily);

export default class App extends Component {
  state = { fontLoaded: false };
  async componentDidMount() {
    await Font.loadAsync({
      IBMPlexSansVar: require('./assets/fonts/IBMPlexSansVar.ttf'),
      'PublicSans-Thin_Regular': require('./assets/fonts/PublicSans-Thin_Regular.ttf'),
    });
    this.setState({ fontLoaded: true });
  }
  render() {
    return this.state.fontLoaded ? (
      <View style={styles.container}>
        <Svg {...{
          fill: '#000',
          height: 200,
          width: '100%',
          viewBox: '-140 0 280 120',
        }}>
          <G fontSize={40} textAnchor="middle">
            <SVGText x={0} y={40} fontFamily="PublicSans-Thin_Regular">
              Hello, world!
            </SVGText>
            <SVGText x={0} y={80} fontFamily="IBMPlexSansVar">
              Hello, world!
            </SVGText>
            <SVGText x={0} y={120}>
              Hello, world!
            </SVGText>
          </G>
        </Svg>
        <Text style={[styles.text, { fontFamily: 'PublicSans-Thin_Regular' }]}>
          Hello, world!
        </Text>
        <Text style={[styles.text, { fontFamily: 'IBMPlexSansVar' }]}>
          Hello, world!
        </Text>
        <Text style={styles.text}>Hello, world!</Text>
      </View>
    ) : null;
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: '#F5FCFF',
  },
  text: {
    fontSize: 56,
    color: '#000',
  }
});

```

**Comment by mattfwood** [comment]
@msand I cloned this repo and tried to run it to see how this was working, but it looks like all the `SVGText` components are rendering as the same font:

![image](https://user-images.githubusercontent.com/22530815/63280045-bd06ab00-c26f-11e9-92ea-c62ac32a2b54.png)

I've been stuck on the same issue for a while and can't seem to figure out a workaround.

**Comment by msand** [comment]
Which repo did you clone? Are you using expo?

**Comment by mattfwood** [comment]
@msand yes, i'm using expo and i cloned the repo you linked above: https://github.com/msand/ExpoTestSvgFonts/blob/master/App.js

**Comment by JoBerkner** [comment]
Any updates on this? I am using the same approach as @nijikokun , perhaps you found a workaround by now?

**Comment by msand** [comment]
Can you make a reproduction?

**Comment by nijikokun** [comment]
@msand dumb question, have you tried the way I was doing it?

**Comment by JoBerkner** [comment]
@msand I am currently not at my office, but I basically did the same approach as @nijikokun a few comments above. My workaround is to use the default react-native `Text` and absolute positioning to have the font I want.

**Comment by dnr** [comment]
I'm running into the same issue, on Android. I'm using Expo 37.0.6 in managed mode, react-native 0.61.4, react-native-svg 11.0.1. Regular react-native Text elements can use a custom ttf font, Svg.Text do not.

I've made the filename, postscript font name, the key in the Font.loadAsync call, and the fontFamily style property all match. I've tried adding and removing `StyleSheet.setStyleAttributePreprocessor('fontFamily', Font.processFontFamily)` (this is not necessary to get fonts working in a Text element). I've tried manually adding the `ExpoFont-` prefix (that's what `processFontFamily` does, sometimes). Everything's still rendering with a system font.

**Comment by gabrielste1n** [comment]
I'm having exactly the same issue as @dnr, seems like it has been an existing problem for at least 2 years

**Comment by sjusaitis** [comment]
For Expo custom fonts, I managed to get it working using `processFontFamily` from the `expo-fonts` package. See https://github.com/expo/expo/issues/1959#issuecomment-780198250

**Comment by Hirbod** [comment]
I can confirm that @sjusaitis comment fixed the issue for me as well!
